src/HotChocolate/Core/src/Types/Types/Pagination/PagingOptions.cs

```diff
@@ -7,5 +7,9 @@ public struct PagingOptions
         public int? MaxPageSize { get; set; }
 
         public bool? IncludeTotalCount { get; set; }
+
+        public bool Forward { get; set; }
```

Comment:
The paging options are meant for both offset and cursor pagination ... if we have some more specialized cases we might want to have an inherited version.

@PascalSenn this again bears the question of how we proceed with pagination.